server.py

```diff
@@ -63,9 +63,6 @@ def parse_args():
                 client_password = input("mypeugeot password: ")
             web.app.myp.connect(client_email, client_password)
         logger.info(str(web.app.myp.get_vehicles()))
-    if args.offline:
-        logger.info("offline mode")
```

Comment:
Why did you remove those lines ?

Reply:
because except if I made a mistake, they are redundant with lines 52, 53, 54.